opensrp-reveal/src/main/assets/json.form/bednet-distribution.json

```diff
@@ -0,0 +1,351 @@
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "subscriberid",
+      "openmrs_entity_id": "163150AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "simserial": {
+      "openmrs_entity_parent": "",
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "simserial",
+      "openmrs_entity_id": "163151AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "phonenumber": {
+      "openmrs_entity_parent": "",
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "phonenumber",
+      "openmrs_entity_id": "163152AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "encounter_location": ""
+  },
+  "step1": {
+    "title": "Bednet Distribution",
+    "display_back_button": "true",
+    "fields": [
+      {
+        "key": "malePopulation",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "normal_edit_text",
+        "hint": "Number of males *",
+        "edit_type": "number",
+        "is_mandatory": true,
+        "v_required": {
+          "value": true,
+          "err": "The male population is required"
+        },
+        "v_numeric": {
+          "value": "true",
+          "err": ""
+        },
+        "v_min": {
+          "value": "0",
+          "err": "There must be greater than or equal to 0"
+        }
+      },
+      {
+        "key": "femalePopulation",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "normal_edit_text",
+        "hint": "Number of females *",
+        "edit_type": "number",
+        "is_mandatory": true,
+        "v_required": {
+          "value": true,
+          "err": "The female population is required"
+        },
+        "v_numeric": {
+          "value": "true",
+          "err": ""
+        },
+        "v_min": {
+          "value": "0",
+          "err": "There must be greater than or equal to 0"
+        }
+      },
+      {
+        "key": "numBedSpaces",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "normal_edit_text",
+        "hint": "Number of bed spaces *",
+        "edit_type": "number",
+        "is_mandatory": true,
+        "v_required": {
+          "value": true,
+          "err": "The number of bed spaces is required"
+        },
+        "v_numeric": {
+          "value": "true",
+          "err": ""
+        },
+        "v_min": {
+          "value": "0",
+          "err": "There must be greater than or equal to 0"
+        }
+      },
+      {
+        "key": "calcNumNetsNeeded",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "hidden",
+        "calculation": {
+          "rules-engine": {
+            "ex-rules": {
+              "rules-file": "bednet-distribution-calculation-rules.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "displayRecommendedNumNets",
```

Comment:
This field needs to dynamically display the number of nets in the label. The field before it does a hidden calculation and it should display something like "5 nets are recommended". I need help figuring out how to do this.

Reply:
use a toast to display that e.g 

```
{
        "key": "toaster_notes",
        "openmrs_entity_parent": "",
        "openmrs_entity": "person_attribute",
        "openmrs_entity_id": "toaster_notes",
        "type": "toaster_notes",
        "text": "This is an information note",
        "text_color": "#1199F9",
        "toaster_type": "info"
      }
```

Reply:
We need to add a calculation to this toaster and the value gets rendered.